proto/config.proto

```diff
@@ -0,0 +1,95 @@
+//package oci.config.bundle;
+package oci;
+
+// Spec is the base configuration for the container.  It specifies platform
+// independent configuration.
+message Spec {
+	// Version is the version of the specification that is supported.
+  optional string Version = 1;
+	// Platform is the host information for OS and Arch.
+  optional Platform Platform = 2; // [default=77];
```

Comment:
If any of these fields are required to be set, can we explicitly state that in the comment?